test/unit/query/reverse.js

```diff
@@ -209,27 +209,6 @@ module.exports.tests.layers = (test, common) => {
 
   });
 
-  test('non-empty array clean.layers should only set non-coarse layers in vs', t => {
-    const clean = {
-      layers: all_layers
-    };
-
-    const query = proxyquire('../../../query/reverse', {
-      'pelias-query': {
-        layout: {
-          FilteredBooleanQuery: MockQuery
-        },
-        view: views,
-        Vars: require('pelias-query').Vars
-      },
-      './reverse_defaults': {}
-    })(clean);
-
-    t.deepEquals(query.body.vs.var('layers').toString(), ['address', 'venue', 'street']);
-    t.end();
-
-  });
-
```

Comment:
Make sure to restore this test and have it check for the new values after the change to the layer calculation :)